KeyVaultCa.Core/KeyVaultCertificateProvider.cs

```diff
@@ -65,13 +69,21 @@ public async Task<IList<X509Certificate2>> GetPublicCertificatesByName(IEnumerab
             int validityInDays,
             bool caCert = false)
         {
+            _logger.LogInformation("Preparing certificate request");
```

Comment:
Consider adding a little bit of context information like the method arguments to the log statement.